README.md

```diff
@@ -15,6 +15,8 @@ Where this charm should be used, is to access one of the following libraries dur
 - [KubernetesServicePatch] - a small library used to patch the Juju auto-created Kubernetes Service
   during the deployment of a sidecar charm to contain the correct ports for an application.
 
+- [JujuTopology] - Used to create and output Juju topologies either from charms, relation data, or parts. 
```

Comment:
This reference style link needs to be defined at the bottom of the file, see line 25-27.

Reply:
Addressed. Thanks!